docs/internal/Ubuntu.md

```diff
@@ -1,6 +1,8 @@
 ### Building and start the app from the sources on Ubuntu Linux
 
-Tested and verified on Ubuntu 18.04.4. The source will be checked out to `~/dev/git/arduino-ide`.
+Tested and verified on Ubuntu 22.04. The source will be checked out to `~/dev/git/arduino-ide`.
```

Comment:
Chicken-egg problem: the `main` still does not support Node.js 18+. Here is what should work. The git checkout part differs

```sh
#!/bin/bash -i

sudo apt update \
&& sudo apt install --no-install-recommends --yes \
  git \
  gcc \
  curl \
  make \
  python3 \
  pkg-config \
  libx11-dev \
  libxkbfile-dev \
  build-essential \
  libsecret-1-dev \
&& wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash \
&& source ~/.bashrc \
&& nvm install 18.17 \
&& nvm use 18.17 \
&& nvm alias default 18.17 \
&& curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add - \
&& echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list \
&& sudo apt update && sudo apt install --no-install-recommends yarn \
&& mkdir -p ~/dev/git/ \
&& rm -rf ~/dev/git/arduino-ide \
&& git clone https://github.com/arduino/arduino-ide.git ~/dev/git/arduino-ide \
&& git -C ~/dev/git/arduino-ide checkout origin/update-infra -b update-infra \
&& yarn --cwd ~/dev/git/arduino-ide \
&& yarn --cwd ~/dev/git/arduino-ide/electron-app rebuild \
&& yarn --cwd ~/dev/git/arduino-ide build \
&& yarn --cwd ~/dev/git/arduino-ide start
```